qiskit/transpiler/passes/layout/vf2_layout.py

```diff
@@ -27,26 +43,50 @@ class VF2Layout(AnalysisPass):
     will be set in the property set as ``property_set['layout']``. However, if no
     solution is found, no ``property_set['layout']`` is set. The stopping reason is
     set in ``property_set['VF2Layout_stop_reason']`` in all the cases and will be
-    one of the following values:
+    one of the values enumerated in ``VF2LayoutStopReason`` which has the
+    following values:
 
         * ``"solution found"``: If a perfect layout was found.
         * ``"nonexistent solution"``: If no perfect layout was found.
+        * ``">2q gates in basis"``: If VF2Layout can't work with basis
 
     """
 
-    def __init__(self, coupling_map, strict_direction=False, seed=None):
+    def __init__(
+        self,
+        coupling_map,
+        strict_direction=False,
+        seed=None,
+        call_limit=None,
+        time_limit=None,
+        properties=None,
+        max_trials=None,
+    ):
         """Initialize a ``VF2Layout`` pass instance
 
         Args:
             coupling_map (CouplingMap): Directed graph representing a coupling map.
             strict_direction (bool): If True, considers the direction of the coupling map.
                                      Default is False.
             seed (int): Sets the seed of the PRNG. -1 Means no node shuffling.
+            call_limit (int): The number of state visits to attempt in each execution of
+                vf2.
+            time_limit (float): The total time limit in seconds to run vf2layout
+            properties (BackendProperties): The backend properties for the backend
```

Comment:
```suggestion
            properties (BackendProperties): The backend properties for the backend. If ``properties.readout_error()`` is available,  it is used to score the layout.
```

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7276/commits/6b82d087c4c94f294940a31e23053c0cf4294aa8